frontend/dist/index.html

```diff
@@ -131,6 +131,7 @@
 
 <body>
   <div id='content'></div>
+  <div id='welcome'><h1>Welcome to Tracim dev</h1></div>
```

Comment:
what is the case to see this Welcome information ?

Reply:
When running through webpack's servdev feature, so it will be displayed only to developers.